data/changelog.txt

```diff
@@ -77,37 +79,39 @@ Adds streetlights to sidewalks; also adds traffic lights, stop lines, and pedest
 Adds Rust Zombies.
 Diversifies venom sources and effects.
 Adds a star fort with working cannon and zombie reenactors.
+Adds Hot Springs location.
+Adds a terrain flag 'NO_SHOOT' to prevent projectiles from striking or going through that tile type.
+Enables using a rope to carry large items.
 
 ## Interface:
-Introduce and migrate most windows to ui_adaptor, which handles redrawing windows when others are closed and the game is resized.
-Crafting GUI - Highlight hidden recipe info.
-Overhaul target selection and aiming UI.
-Display missing and obsolete mods for current world.
-Require grenades and other explosives to be wielded before activation.
-Add bigger mini-maps to all sidebar layouts.
-Add keycode support to uilist.
-Allow disabling robots by 'e'xamining them.
-Warn if friendly creature or non-hostile NPC enters line of fire.
+Introduces, and migrates most windows to, ui_adaptor, which handles redrawing windows when others are closed and the game is resized.
+Highlights hidden recipe info in the crafting GUI.
+Overhauls target selection and aiming UI.
+Displays missing and obsolete mods for current world.
+Requires grenades and other explosives to be wielded before activation.
+Adds bigger mini-maps to all sidebar layouts.
+Adds keycode support to uilist.
+Allows attempting to disable robots by 'e'xamining them.
+Warns if a friendly creature or non-hostile NPC enters the line of fire.
 Replaces the CAL/kg column of the (E)at menu with a more meaningful Satiety column.
 Adds a Mod to use MSX characters and monsters with the Ultica tileset.
 Show missing proficiency failure rates for crafting.
 Display contained items like 'water (plastic bottle)' instead of 'plastic bottle of water'.
```

Comment:
```suggestion
Shows crafting failure rates due to missing proficiencies.
Displays contained items like 'water (plastic bottle)' instead of as 'plastic bottle of water'.
```